TESTING.md

```
@@ -0,0 +1,487 @@
+
+        // Test with Pinocchio backend - Failure case
+        #[cfg(feature = "pinocchio")]
+        {
+            let key = [1u8; 32];
+            let owner = [2u8; 32];
+            let account = create_test_account_pinocchio(key, owner, 16, false, false, false); // Not writable
+            let result = function_under_test(&account);
+            assert_eq!(result.unwrap_err(), AccountError::AccountNotMutable);
+        }
+    }
+}
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)